staging/src/k8s.io/client-go/rest/config.go

```diff
@@ -476,15 +476,26 @@ func DefaultKubernetesUserAgent() string {
 		adjustCommit(version.Get().GitCommit))
 }
 
+func getPath(path string) string {
+	if gruntime.GOOS == "windows" {
+		// This env variable will only be set on HostProcess containers and will be blank for other windows containers
+		// https://kubernetes.io/docs/tasks/configure-pod-container/create-hostprocess-pod/#volume-mounts
+		sandbox := os.Getenv("CONTAINER_SANDBOX_MOUNT_POINT")
+		if sandbox != "" {
+			return fmt.Sprintf("%s%s", sandbox, path)
```

Comment:
Might be better to `filepath.Join` here, even though `C:\C\12345678abcdefg\/var/run/secrets/kubernetes.io/serviceaccount/token` is a valid path haha.